app/views/flood_risk_engine/enrollments/steps/_local_authority_address.html.erb

```diff
@@ -1 +1,34 @@
-local_authority_address
+<%= t('.postcode_hint') %>
+<p>
+  <%= form.post_code %>
+  <%= link_to t('.change'),
+              enrollment_step_path(form.enrollment, form.enrollment.previous_step),
+              class: 'change-postcode-button'
+  %>
+</p>
+
+<%= f.hidden_field :post_code, value: form.post_code %>
+
+<label class="form-label" for="address_match_selection">
+  <%= t('.select_from_list_label') %>
+</label>
+
+<% form.address_list.unshift({ moniker: t('.num_addresses_found', num:  form.address_list.count) }) %>
```

Comment:
The list count should not be added to the list in this way - use the select form helper's prompt option